external-plugins/referee/server/tooManyRetestsComment.gomd

```diff
@@ -0,0 +1,48 @@
+{{- /*
+This file is part of the KubeVirt project
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Copyright the KubeVirt authors.
+*/ -}}
+{{- /* gotype: kubevirt.io/project-infra/external-plugins/referee/server.TooManyRequestsData */ -}}
+✋🧢
+
+/hold
+
+Dear @{{ $.Author }}
+
+⚠️ this pull request exceeds the number of retests that are allowed per individual commit.
+
+🔎 Please check that the changes you committed are fine and that there are no infrastructure issues present!
+
+<details>
+Checklist:
+
+* [ ] your changes compile
+* [ ] your tests succeed **locally**, i.e. for [k/kubevirt](https://github.com/kubevirt/kubevirt) see
+  * [getting-started - testing](https://github.com/kubevirt/kubevirt/blob/main/docs/getting-started.md#testing) and
+  * [CONTRIBUTING.md - testing](https://github.com/kubevirt/kubevirt/blob/main/CONTRIBUTING.md#testing)
+* [ ] no linting errors
+* [ ] no recurring e2e test errors, i.e. in the `pull-kubevirt-check-tests-for-flakes` lane
+* [ ] no infrastructure issues, i.e.
```

Comment:
I think this point is more relevant to us - I am not sure we can expect regular contributors to be aware of infrastructure issues. This might be unfair to suggest that they try to figure out if there is an infra issue?

Reply:
I mentioned this rather for completeness and awareness. In any case we are pinged on the PR, so we might be able to chime in pretty fast in case this actually is happening.

Reply:
Ok that is fair enough.